style/index.js

```diff
@@ -372,7 +375,9 @@ function types(...type) {
   return type.map((x) => `[type] = '${x}'`).join(' or ');
 }
 
-const mapnikConfig = generateFreemapStyle();
+const mapnikConfig = generateFreemapStyle()
+  .replace(/&lt;&lt;/g, "<")
+  .replace(/&gt;&gt;/g, ">");
```

Comment:
PS: Namiesto toho hacku je lepsie pridat "extension" ako to robim v uvode. Pozriem sa na to.